client/my-sites/checkout/composite-checkout/components/checkout-terms-item.tsx

```diff
@@ -0,0 +1,53 @@
+import { Gridicon } from '@automattic/components';
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+
+const Wrapper = styled.div`
+	padding-left: 24px;
+	position: relative;
+	font-size: 12px;
+
+	> svg {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 16px;
+		height: 16px;
+
+		.rtl & {
+			left: auto;
+			right: 0;
+		}
+	}
+
+	p {
+		font-size: 12px;
+		margin: 0;
+		word-break: break-word;
+	}
+`;
+
+const CheckoutTermsItem = ( {
+	children,
+	className,
+	onClick,
+	wrapChildren = true,
```

Comment:
Neat solution, although an optional boolean that defaults to true feels like a less-than-ideal API to me. What about renaming it to `noWrapChildren` or something and then defaulting to false? That way you can opt-out (which ultimately is what this does) by just specifying the bare prop like this:

```js 
<CheckoutTermsItem noWrapChildren>...</CheckoutTermsItem>
```

which feels more semantic to me than:

```js 
<CheckoutTermsItem wrapChildren={false}>...</CheckoutTermsItem>
```

Reply:
I see your points but I'm on the fence. I'm not a fan of `noWrapChildren` because it introduces a negation, which makes its name a little bit more difficult to reason about than just `wrapChildren` ([this post](https://davej.io/2018/05/negative-naming.html) better explains this). Another problem with this approach is that it will force us to add `noWrapChildren` in a lot of places whereas we would need to declare `wrapChildren` once. Maybe `withoutParagraph` could be a good middle-ground 😀?

Reply:
Fair point! I went with `prewrappedChildren` and inverted the truthy/falsy logic. WDYT, is that an acceptable prop name?

Reply:
I should also mention that I looked at the possibility of removing this prop entirely in favor of inspecting the `children` to see if they contain block elements or not. Turns out that this very quickly gets complex once you have block elements nested within `Fragment`s (which we have), so I abandoned this approach.

Reply:
> I'm not a fan of noWrapChildren because it introduces a negation, which makes its name a little bit more difficult to reason about than just wrapChildren

I agree! If `wrapChildren` defaulted to false, then I think it would be better, but I was trying to come up with a way to keep wrapping as the default without having to have a truthy optional prop default.

> Fair point! I went with prewrappedChildren and inverted the truthy/falsy logic. WDYT, is that an acceptable prop name

👍  Sounds fine to me. Ultimately someone using this component needs to understand what "wrapped" means anyway, but at least this way there might be one fewer surprises!

> I should also mention that I looked at the possibility of removing this prop entirely in favor of inspecting the children to see if they contain block elements or not. Turns out that this very quickly gets complex once you have block elements nested within Fragments (which we have), so I abandoned this approach.

Yes, learning about children is very tricky. [It can be done](https://github.com/Automattic/wp-calypso/blob/41b943648e2f9b45000af1706e843ba7212b235c/packages/composite-checkout/src/components/checkout-steps.tsx#L162-L163) but I've had the same problems you describe here. Avoiding it as you've done is a good idea.

Reply:
Since `prewrappedChildren` is a boolean, I think I would go with `isPrewrappedChildren`.

Reply:
Done! I've renamed the prop